src/Virtual Car Dealer/r6/scripts/VirtualCarDealer/core/CarDealer-System.reds

```diff
@@ -5,12 +5,17 @@ import CarDealer.Classes.AutofixerItemData
 import CarDealer.Config.CarDealerConfig
 import CarDealer.Utils.CarDealerLog
 
+@if(ModuleExists("VehiclePersistence.System"))
+import VehiclePersistence.System.PersistentVehicleSystem
+
 public class PurchasableVehicleSystem extends ScriptableSystem {
 
   private let m_storeVehicles: array<ref<PurchasableVehicleBundle>>;
 
   private let m_vehicleSystem: ref<VehicleSystem>;
 
+  private let m_gameInstance: GameInstance;
```

Comment:
ScriptableSystem has `this.GetGameInstance()` access, better to use it I think instead of storing it in a new field

Reply:
Ah nice, didn't know that - will test it out and update